src/components/dictionaryConcepts/components/ViewConceptMappings.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+import ViewMappingsModal from './ViewMappingsModal';
+
+class ViewConceptMappings extends Component {
+  constructor(props) {
```

Comment:
I think a constructor is unnecessary. `state = {modal : false} ` can be enough.

Reply:
This makes sense. Fixed it.